x-pack/plugins/monitoring/public/application/index.tsx

```diff
@@ -111,6 +112,13 @@ const MonitoringApp: React.FC<{
                   />
 
                   {/* Beats Views */}
+                  <RouteInit
+                    path="/beats/beat/:instance"
+                    component={BeatsInstancePage}
+                    codePaths={[CODE_PATH_ELASTICSEARCH]}
```

Comment:
Should this be `CODE_PATH_BEATS` (see [angular component](https://github.com/elastic/kibana/blob/master/x-pack/plugins/monitoring/public/views/beats/beat/index.js#L24)) ? It even looks like it's not used by the [endpoint](https://github.com/elastic/kibana/blob/master/x-pack/plugins/monitoring/server/routes/api/v1/beats/beat_detail.js#L26-L31) but maybe an intermediary component ?

Reply:
ooh, nice catch!